interfaces/apparmor/spec.go

```diff
@@ -235,6 +235,43 @@ func (spec *Specification) UpdateNSIndexOf(snippet string) (idx int, ok bool) {
 	return spec.updateNS.IndexOf(snippet)
 }
 
+func (spec *Specification) emitLayout(si *snap.Info, layout *snap.Layout) {
```

Comment:
we use it elsewhere, but is very rare that to use si short var name for snap.Info, it is mostly used for SideInfo. oh well. I should probably remove the few places that do this, but is not a concern for this PR in itself.

Reply:
I'll change it to snapInfo for this PR?

Reply:
This was not done yet but I assume it could be a followup given that there are more places where this is needed